indico/modules/events/abstracts/forms.py

```diff
@@ -192,7 +192,8 @@ class EditEmailTemplateRuleForm(IndicoForm):
     title = StringField(_("Title"), [DataRequired()])
     rules = EmailRuleListField(_("Rules"), [DataRequired()])
     stop_on_match = BooleanField(_("Stop on match"), [DataRequired()], widget=SwitchWidget(), default=True,
-                                 description=_("Do not evaluate any other rules once this one matches."))
+                                 description=_("If any of the rules from this email template match, do not "
```

Comment:
What about this? `If this notification is triggered do not send any other after`
/cc @ThiefMaster @pferreir